samples/Hystrix.Dotnet.Samples.AspNetCore/Controllers/HelloController.cs

```diff
@@ -19,10 +19,19 @@ public HelloController(IHystrixCommandFactory hystrixCommandFactory)
         
         [HttpGet]
         public async Task<IActionResult> Get()
-        { 
+        {
+            var firstCall = true;
+
             var result = await hystrixCommand.ExecuteAsync(
                 async () =>
                 {
+                    if(firstCall)
+                    {
+                        // On the first call we throw an exception to be able to illustrate Retry.
+                        firstCall = false;
+                        throw new InvalidOperationException();
```

Comment:
Shouldn't that be the responsibility of the user of Hystrix? I mean I definitely don't want to promote blindly overusing retrial, that's why I set the default to 0, but for example in my scenario I have to retry when a timeout happens., so only retrying on network errors wouldn't be enough for me.  
I also checked the implementation of Polly, and they also simply catch any exception: https://github.com/App-vNext/Polly/blob/master/src/Polly.Shared/Retry/RetryEngine.cs#L38

I'm afraid if we make it too restrictive and specific, for example if we only enable retrial in case of certain network exception types, it won't be generic enough and people won't use Hystrix, but just rather implement their own retrial mechanism (or use Polly). What do you think?